projects/saturn/src/main/java/io/fairspace/saturn/controller/enums/CustomMediaType.java

```diff
@@ -0,0 +1,13 @@
+package io.fairspace.saturn.controller.enums;
+
+public enum CustomMediaType {
+    ;
```

Comment:
```suggestion
```

Reply:
it is a must, otherwise enum would not be possible to compile